scripts/deeplink-to-emusim.sh

```
@@ -0,0 +1,93 @@
+
+echo -e "\n\033[1mSelect the target device:\033[0m\n"
+PS3=$'\nEnter the number corresponding to your choice: '
+options=("iOS Simulator" "Android Emulator")
+
+select device in "${options[@]}"; do
+  if [ "$device" == "iOS Simulator" ]; then
+    echo -e "\nSending deep link to iOS simulator: $selectedLink\n"
+    xcrun simctl openurl booted "$selectedLink"
+    break
+  elif [ "$device" == "Android Emulator" ]; then
+    if [ $(echo "$android_emus" | wc -l) -eq 1 ]; then
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)